spec/models/profile_spec.rb

```diff
@@ -98,7 +98,12 @@
 
       expect(profile.activated_at).to be_nil
       expect(profile.active).to eq(false)
-      expect(profile.deactivation_reason).to eq('in_person_verification_pending')
+
+      # can be deleted when column is removed
+      expect(profile.deactivation_reason).to be_nil
```

Comment:
The column is still used for password_reset, encryption_error, and verification_canceled, so it won't be removed.

Reply:
removed the comment